core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -456,6 +463,8 @@ object Chunk {
   }
   object Doubles { def apply(values: Array[Double]): Doubles = Doubles(values, 0, values.length) }
 
+  implicit def fs2EqForChunk[A: Eq: ClassTag]: Eq[Chunk[A]] = Eq.by(_.toArray.toVector)
```

Comment:
This was necessary only for testing, and perhaps should accordingly move into TestUtil.  The question is whether it's wrong.

Reply:
I think this should probably go in `TestUtil` -- or if not, then at least drop the `toArray` so we can drop the `ClassTag` constraint.

Reply:
Since a Chunk is strict and finite, I don't see a lot of harm.  I'd overlooked `toVector`, which removes the `ClassTag` constraint.

If I were to argue for its banishment to `TestUtil`, I'd say it's because it's inconsistent with (more generous than) universal equality.

Reply:
I guess we could do better by comparing sizes and then short circuit on the first unequal A.  We don't need to materialize a vector.